src/cli/extensions/helpers.js

```diff
@@ -0,0 +1,281 @@
+import chalk from 'chalk';
+import resolve from 'resolve';
+import { isString } from 'lodash';
+import semver from 'semver';
+
+import { registerHooks, getHooks, setHooks } from '../../hooks';
+import { registerActions, getActions, setActions } from '../../hooks/actions';
+import { merge } from '../../configuration';
+import { warningLabel, feedbackMessage } from '../../helpers/style';
+import ExtensionError from './error';
+
+export function runPostInits(initialState) {
+    return initialState.postInits.reduce(
+        (state, postInit) =>
+            manageRocObject(
```

Comment:
Yes. Doing this later is okay.
We'll get back to this and have a chat before any other poential larger change-sets. :)